test/images/netexec/netexec.go

```diff
@@ -283,6 +310,9 @@ func startUDPServer(udpPort int) {
 	defer serverConn.Close()
 	buf := make([]byte, 1024)
 
+	log.Printf("Started udp server")
+	// Start responding to readiness probes.
+	serverReady.set()
```

Comment:
can you add a defer that resets serverReady.
that way, if udp server dies, the health check will fail.